### PR TITLE
Lots of f strings and things

### PR DESCRIPTION
**Pull request by TrevorEdris**
## Proposed Changes

- Add lots of f strings
- A few other helpful optimizations

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -24,10 +25,6 @@ def main():
         else:
             print("Invalid option try again")
```
Comment (TDI00-7) [inline main.py:27]: Modify this line

File: writer.py [hunk 2]
```diff
@@ -59,12 +62,9 @@ def csv_row_editer(csv_file):
             for i in range(len(csv_list)):
                 read_file.writerow(csv_list[i])
 
+
 def csv_column_editer(csv_file):
-    csv_list = []
```
Comment (TDI00-7) [inline writer.py:63]: why change this line

**Review by TDI00-7 — CHANGES_REQUESTED** [review]
I like the formatting changes.

File: main.py (hunk 1 above)
Comment (TrevorEdris) [reply main.py:27]: i'm not gonna

File: writer.py (hunk 2 above)
Comment (TrevorEdris) [reply writer.py:63]: because it's better this way

**Review by TDI00-7 — APPROVED** [review]

**Pull request merged**